src/librustc_yk_sections/emit_sir.rs

```diff
@@ -57,6 +57,8 @@ struct ConvCx<'a, 'tcx, 'gcx> {
     mir: &'a Body<'tcx>,
     /// The DefId of the above MIR.
     def_id: DefId,
+    /// Extra DefIds encountered during the lowering.
+    extra_def_ids: DefIdSet,
```

Comment:
What does "extra" mean in this context?

Reply:
They are DefIds that we will need to process later.

Reply:
"extra" doesn't seem a very descriptive name. "Nested" or something similar might be clearer?

Reply:
Hrm. It doesn't feel quite right either. How about `child_def_ids` and a comment explaining a little better?

Reply:
Sounds like a plan!